test/indentation_test.js

```diff
@@ -52,4 +52,16 @@ describe('indentation', () => {
       });
     `);
   });
+
+  it('handles indented lambda as arg to inline lambda', () => {
+    check(`
+    Q -> P ->
```

Comment:
Sample code should be indented one more level, so one indentation level past the `check`.